app/static/js/initialize_map.js

```diff
@@ -0,0 +1,155 @@
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+//Initial map center coordinates
+INITIAL_CENTER_LAT = 39.952;
```

Comment:
Is hardcoding center coordinates necessary if we are just going to auto-recenter the map?